DataFormats/SiStripClusterSoA/interface/SiStripClustersSoA.h

```diff
@@ -0,0 +1,30 @@
+
+#ifndef DataFormats_SiStripClusterSoA_interface_SiStripClustersSoA_h
+#define DataFormats_SiStripClusterSoA_interface_SiStripClustersSoA_h
+
+#include "DataFormats/Common/interface/StdArray.h"
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+
+namespace sistrip {
+  const static auto maxStripsPerCluster = 768;
+  using clusterADCsColumn = edm::StdArray<uint8_t, maxStripsPerCluster>; /*768*/
```

Comment:
Can you make this uppercase, like other class names ?
```suggestion
  using ClusterADCsColumn = edm::StdArray<uint8_t, maxStripsPerCluster>; /*768*/
```

Reply:
Temporarily fixed in commit afa7fc77a21f0a1d982632504f2e727f3ade872c
(temporarily = we agreed to change this dinamically according to maxClusterSize from configuration. This will be implemented in a later commit)

Reply:
I would prefer that the cluster size is kept dynamic following my suggestion in https://github.com/cms-sw/cmssw/pull/47629/files/9fe439a4025eae782fdfbef706167dda573653f4#r2014482802 .

As far as I can tell, that should also be close to the minimal memory usage as possible.